source/common/conn_pool/conn_pool_base.h

```diff
@@ -241,6 +241,9 @@ class ConnPoolImplBase : protected Logger::Loggable<Logger::Id::pool> {
   // Clients that are not ready to handle additional streams because they are CONNECTING.
   std::list<ActiveClientPtr> connecting_clients_;
 
+  void onUpstreamReady();
+  Event::SchedulableCallbackPtr upstream_ready_cb_;
```

Comment:
nit: should be private.

Reply:
i have no idea why that wasn't. i wonder if i moved it before fixing something :/